CMakeLists.txt

```diff
@@ -1,46 +1,17 @@
 cmake_minimum_required(VERSION 3.1)
-project(bspline)
+project(splx)
 
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -pedantic -Wextra")
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 14)
 set(CMAKE_BUILD_TYPE Debug)
 
-find_package(Eigen3 REQUIRED)
-
-
-include_directories(
-  ${PROJECT_SOURCE_DIR}/include
-  ${EIGEN3_INCLUDE_DIR}
-  ${QPOASES_INCLUDE_DIR}
-)
-
-
-add_executable(
-bsplinetest
-test/testbspline.cpp
-)
+add_compile_options(-std=c++14)
```

Comment:
Doesnt set(CMAKE_CXX_STANDARD 14) do this already?

Reply:
yes, It was a leftover from the compile trials, removed